src/lib.rs

```diff
@@ -433,7 +433,8 @@ impl Drop for ArcCStr {
 impl PartialEq for ArcCStr {
     /// Equality for two `ArcCStr`s.
     ///
-    /// Two `ArcCStr`s are equal if their underlying strings are equal.
+    /// Two `ArcCStr`s are equal if they point to the same underlying string or
+    /// their underlying strings are equal
```

Comment:
I don't think we need this clarification. If the pointers are equal, the underlying strings must also be equal.

Reply:
Fair enough, reverted.